cpp/tests/io/parquet_test.cpp

```diff
@@ -4215,4 +4268,64 @@ TEST_F(ParquetReaderTest, EmptyColumnsParam)
   EXPECT_EQ(result.tbl->num_rows(), 0);
 }
 
+TEST_F(ParquetReaderTest, BinaryAsStrings)
+{
+  std::vector<const char*> strings{
+    "Monday", "Wednesday", "Friday", "Monday", "Friday", "Friday", "Friday", "Funday"};
+  const auto num_rows = strings.size();
+
+  auto seq_col0 = random_values<int>(num_rows);
+  auto seq_col2 = random_values<float>(num_rows);
+  auto validity = cudf::detail::make_counting_transform_iterator(0, [](auto i) { return true; });
```

Comment:
We have an app for that now: [`cudf::test::iterators::no_nulls()`](https://github.com/rapidsai/cudf/blob/branch-22.08/cpp/include/cudf_test/iterator_utilities.hpp#L116), which boils down to `thrust::make_constant_iterator(true)`.